src/Directory.Build.props

```diff
@@ -5,6 +5,7 @@
     <VersionPrefix>1.4.16</VersionPrefix>
     <PackageTags>akka;actors;actor model;Akka;concurrency;persistence;eventsource;redis</PackageTags>
     <PackageIconUrl>http://getakka.net/images/AkkaNetLogo.Normal.png</PackageIconUrl>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
```

Comment:
The `README.md` is included in the package in line 63 of this file so it could be referenced. 

I noticed that in the MultNodeTestRunner changes were done for the Icon. Is that something that should be done for this repo as well? It current references the icon from the getakka site (Line 7 of this file)
https://github.com/akkadotnet/Akka.MultiNodeTestRunner/pull/201/files#diff-5c1900dca99df01c3ddc3a884282164b04deee76e79b176e2d34cfa47f3b8907R23